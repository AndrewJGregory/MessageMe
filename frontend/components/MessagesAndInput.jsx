import React from "react";

const MessagesAndInput = () => {
  return <section className="messages">Messages and input go here</section>;
};

export default MessagesAndInput;
