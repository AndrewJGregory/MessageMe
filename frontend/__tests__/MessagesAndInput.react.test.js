import React from "react";
import MessagesAndInput from "../components/MessagesAndInput";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<MessagesAndInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
