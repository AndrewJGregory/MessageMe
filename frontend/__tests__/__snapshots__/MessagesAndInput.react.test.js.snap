// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="messages"
>
  <div
    className="under-construction-info"
  >
    <p
      className="other-text"
    >
      Most recent completed feature: searching users 
      <br />
      Next up: direct messages 
      <br />
       This is currently under construction. While you're here, check out what else I've done:
    </p>
    <ul
      className="links"
    >
      <li>
        <a
          href="https://www.linkedin.com/in/andrewjgregoryajg/"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-linkedin"
          />
        </a>
      </li>
      <li>
        <a
          href="http://www.andrewjgregoryajg.com"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-globe"
          />
        </a>
      </li>
      <li>
        <a
          href="https://github.com/AndrewJGregory/"
        >
          <i
            aria-hidden="true"
            className="fa fa-2x fa-github"
          />
        </a>
      </li>
    </ul>
  </div>
  <div
    className="under-construction-img"
  />
</section>
`;
